sahi/model.py

```diff
@@ -66,7 +65,7 @@ def __init__(
         self._object_prediction_list_per_image = None
 
         # automatically set device if its None
-        if not (self.device):
+        if not (self.set_device):
```

Comment:
yanlis olmus device set edilmis mi diye bakacagiz burada:
```suggestion
        if not (self.device):
```